### PR TITLE
fix(getters): incorrect content projection

### PR DESCRIPTION
**Pull request by lksmsr**

### CODE REVIEW

File: src/utils/getters.ts [hunk 1]
```diff
@@ -59,6 +60,52 @@ export function findNavigationItemInNavigationData(
...
+  $store: Store<RootState>,
+  $fsxaApi: FSXAApi,
+  locale: string,
+  currentPath?: string,
+) {
+  const navigationData = $store.state.fsxa.navigation;
+  if (!navigationData || !currentPath) return;
+  const path = decodeURIComponent(currentPath || "");
+  if (findNodeInSeoRouteMap(path, navigationData)) return;
+  if (getStoredItem($store, currentPath)) return;
```
Comment (deangite) [inline src/utils/getters.ts:102]: please you curly brackets
```
const navigationData = $store.state.fsxa.navigation;

  if (!navigationData || !currentPath) {
    return;
  }

  const path = decodeURIComponent(currentPath || "");

  if (
    findNodeInSeoRouteMap(path, navigationData) ||
    getStoredItem($store, currentPath)
  ) {
    return;
  }
  ```

File: src/utils/getters.ts [hunk 2]
```diff
@@ -72,7 +119,15 @@ export async function triggerRouteChange(
   globalSettingsKey?: string,
 ): Promise<string | null> {
   if (!params.locale || params.locale === currentLocale) {
-    if (params.route) return params.route;
+    if (params.route) {
+      await fetchDatasetIfMissing(
```
Comment (deangite) [inline src/utils/getters.ts:123]: please change the name of function to `fetchMissingDataset`

**Review by deangite — CHANGES_REQUESTED** [review]

**Review by henczi-espirit — CHANGES_REQUESTED** [review]
This PR contains an imcomplete implementation

**Comment by henczi-espirit** [comment]
issue will be resolved by a different PR

**Pull request closed without merging**